src/composables/useLock.ts

```diff
@@ -64,6 +65,20 @@ export function useLock({ enabled = true }: Options = {}) {
       : '0'
   );
 
+  const poolShares = computed(() => {
+    if (!lockPool.value) return bnum(0);
+    return bnum(lockPool.value.totalLiquidity).div(lockPool.value.totalShares);
+  });
```

Comment:
Please rename this `bptPrice` or something similar, because `poolShares` isn't accurate.